src/store/modules/chat/index.js

```diff
@@ -587,7 +574,8 @@ const actions = {
    * Push array of messages and sort by senderId.
    * @param {Message[]} messages Array of messages
    */
-  pushMessages({ commit, rootState }, messages) {
+  pushMessages({ commit, rootState, dispatch }, messages) {
+    dispatch('reInitCommands', messages)
```

Comment:
Are you sure you need the raw messages? Can you use already normalized messages (see 581 line)?